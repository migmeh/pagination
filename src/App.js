import React, { Component } from "react";
import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
import "bootstrap/dist/css/bootstrap.min.css";
import "./App.css";

import TutorialsList from "./components/list.component";

class App extends Component {
  render() {
    return (
        <Router>
          <div>
            <nav className="navbar navbar-expand navbar-dark bg-dark">
                <h1 className='mytitle'>Welcome to my website</h1>
            </nav>

            <div className="container mt-3">
              <Switch>
                <Route exact path={["/", "/tutorials"]} component={TutorialsList} />

              </Switch>
            </div>
          </div>
        </Router>
    );
  }
}

export default App;
