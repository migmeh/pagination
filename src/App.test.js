import { render, screen } from '@testing-library/react';
import App from './App';
import { shallow } from 'enzyme';
import React from "react";


describe('component app', ()=>{
test('renders learn react link', () => {
  render(<App />);
  const linkElement = screen.getByText("Welcome to my website");
  expect(linkElement).toBeInTheDocument();
});




  it('render', ()=>{
    const wrapper = <App />;
    expect(wrapper).toMatchSnapshot();
  });
});
