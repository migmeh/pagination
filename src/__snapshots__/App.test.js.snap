// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component app render 1`] = `<App />`;
